python/vyos/ipsec.py

```diff
@@ -83,6 +83,35 @@ def get_vici_sas_by_name(ike_name: str, tunnel: str) -> list:
         raise ViciCommandError(f'Failed to get SAs')
 
 
+def get_vici_connection_by_name(ike_name: str) -> list:
+    """
+    Find sas by IKE_SA name and/or CHILD_SA name
```

Comment:
Nitpick: "sas" as an all-lowercase word looks jarring — should be "SAs" since it's an abbreviation.